src/builtin-templates/hosted-payment-card-form.js

```diff
@@ -1,29 +1,42 @@
 const BASE_URL = 'https://vault.automationcloud.net/forms/index.html';
 const DEFAULT_OPTIONS = {
-    fields: 'pan,name,expiry-select,cvv',
-    brands: 'visa,mastercard,amex',
-    validateOnInput: 'on',
-    css: 'https://ubio-application-bundle-dummy-server.glitch.me/style.css'
+    fields: ['pan', 'name', 'expiry-select', 'cvv'],
+    brands: ['visa', 'mastercard', 'amex']
 };
 
+function formatFields(fields) {
+    const formatted = [];
+
+    for (const element of fields) {
+        formatted.push(typeof element === 'string' ? element : `${element.field}_${element.name}`);
+    }
+
+    return formatted.join(',');
+}
+
+function formatBrands(brands) {
+    return brands.join(',');
+}
+
 export default (otp, formOptions = {}, formStyles = {}) => {
-    const options = {
-        ...DEFAULT_OPTIONS,
-        ...formOptions
-    };
+    if (!formOptions.css) {
+        throw new Error('css is a required field option.');
```

Comment:
hm... it's not actually mandatory) but probably it makes sense

Reply:
I've made it mandatory here since the glitch URL shouldn't be considered immutable. Our userland templates do hard code this though.